frontend/src/Dashboard/CrossingListPage/CrossingList.js

```diff
@@ -40,15 +47,23 @@ class CrossingList extends React.Component {
     }
 
     return (
-      <div className='CrossingListContainer'>
-        {crossings.map(crossing => 
-          <CrossingListItem
-            crossing={crossing}
-            reasons={statusReasons} 
-            durations={statusDurations}
-            hidden={!crossingIdsToShow.includes(crossing.id)} />
-        )}
-      </div>
+      <ContainerQuery query={containerQuery}>
+        {(params) => {
+          const cqClassName = classnames(params, 'CrossingListItem');
+          return (
+            <div className='CrossingListContainer'>
+              {crossings.map(crossing => 
+                <CrossingListItem
+                  crossing={crossing}
+                  reasons={statusReasons} 
+                  durations={statusDurations}
+                  hidden={!crossingIdsToShow.includes(crossing.id)} 
+                  cqClassName={cqClassName} />
```

Comment:
Stylistic thing that you can take or leave, but I like moving the `/>` to its own line and indented at the same level as the opening tag. This saves the messy diff lines and also shows you clearly where the component closes.